app/views/works/top.html.erb

```diff
@@ -0,0 +1,29 @@
+<section class="row spotlight">
+  <div>
+    <h3>
+      <span class="media-spotlight-header">Media Spotlight: </span>
+      <%= link_to @top.title %>
+      by
+      <%= @top.creator %>
+    </h3>
+    <p><%= @top.votes.count %> votes - <%= @top.description %></p>
+  </div>
+</section>
+
+<hr class="root-hr">
+
+<section class="top-ten-container">
+  <% @categories.each do |category| %>
+  <section class="top-ten">
+    <h3>Top <%= category.name.capitalize %>s</h3>
+    <ul>
+    <% @works.where(category: category).top(10).each do |work| %>
+    <li>
```

Comment:
The query on line 20 definitely counts as doing business logic in the view. Instead, you might write an instance method on the `Category` model that produces the same information.